### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by B3r3z**

### CODE REVIEW

File: DockerRun.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+#! /bin/bash
```
Comment (kecajjo) [inline DockerRun.sh:1]: finish this script before merging and move it to docker folder

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+add_compile_options(-Wall -g -Wextra -Wpedantic)
+
+include_directories()
+
+# add the executable
+add_executable(My_chess_Game
+                main.cpp
+)
```
Comment (kecajjo) [inline CMakeLists.txt:15]: Move this file to src folder

File: run.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cmake . -B build
+cd build 
+make
```
Comment (kecajjo) [inline run.sh:4]: move file to src folder

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        plantuml \
+        clang-format \
+        git && \
+        rm -rf /var/lib/apt/lists/*
+    
+USER developer
+    RUN mkdir -p /home/developer/Project
+    WORKDIR /home/developer/Project
+
+ENTRYPOINT [ "/bin/bash", "-c" ]
```
Comment (kecajjo) [inline Dockerfile:19]: Create docker folder and move file to it

**Review by kecajjo — CHANGES_REQUESTED** [review]
Create proper folder structure and mentioned in the comments, will help later on
you can also add some README.md and license (MIT for example)

File: DockerRun.sh (hunk 1 above)
Comment (B3r3z) [reply DockerRun.sh:1]: ok

File: CMakeLists.txt (hunk 2 above)
Comment (B3r3z) [reply CMakeLists.txt:15]: fixed

File: run.sh (hunk 3 above)
Comment (B3r3z) [reply run.sh:4]: fixed

File: Dockerfile (hunk 4 above)
Comment (B3r3z) [reply Dockerfile:19]: fixed

File: DockerRun.sh (hunk 1 above)
Comment (kecajjo) [reply DockerRun.sh]: done

**Review by kecajjo — APPROVED** [review]

**Pull request merged**